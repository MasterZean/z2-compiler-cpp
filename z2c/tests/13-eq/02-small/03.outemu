Bool{false}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
Bool{true}
