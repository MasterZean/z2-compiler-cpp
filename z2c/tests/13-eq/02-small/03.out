-100 == (bool)(-100);
-100 == -100;
-100 == (int8)(156u);
-100 == -100;
-100 == (int8)(65436u);
-100 == -100;
-100 == (int8)(4294967196u);
-100 == -100ll;
-100 == (int8)(18446744073709551516ull);
-100 == -100.0f;
-100 == -100.0;
-100 == (int8)(4294967196u);
